go-controller/pkg/ovn/master.go

```diff
@@ -772,6 +772,23 @@ func (oc *Controller) deleteNode(nodeName string, nodeSubnet, joinSubnet *net.IP
 		return fmt.Errorf("Failed to clean up node %s gateway: (%v)", nodeName, err)
 	}
 
+	chassis_name, stderr, err := util.RunOVNSbctl("--data=bare", "--no-heading",
+		"--columns=name", "find", "Chassis",
+		"hostname="+nodeName)
+	if err != nil {
+		klog.Errorf("Failed to get chassis name for logical switch %s: stderr: %q, error: %v",
+			nodeName, stderr, err)
+		return err
+	}
+
+	_, stderr, err = util.RunOVNSbctl("--if-exist", "chassis-del", chassis_name)
```

Comment:
probably have to check if `chassis_name != ""`. I don't think its an error if you get zero results.

Reply:
👍